source/VisTrace.cpp

```diff
@@ -228,6 +229,53 @@ LUA_FUNCTION(RT_GenerateMIPs)
 	return 0;
 }
 
+LUA_FUNCTION(RT_Load)
+{
+	LUA->CheckType(1, RenderTarget::id);
+	LUA->CheckType(2, Type::String);
+
+	RenderTarget* pRt = *LUA->GetUserType<RenderTarget*>(1, RenderTarget::id);
+
+	if (!pRt->IsValid()) LUA->ThrowError("Invalid render target");
+
+	const char* filename = LUA->GetString(2);
+	if (!filename) {
```

Comment:
You don't need this check, if the type was a string and this returned a nullptr, the engine's Lua API itself is broken